docs/source/conf.py

```diff
@@ -393,6 +393,7 @@
     ("joblib.html", "https://ml.dask.org/joblib.html"),
     ("setup.html", "https://docs.dask.org/en/latest/setup.html"),
     ("ec2.html", "https://docs.dask.org/en/latest/setup/cloud.html"),
+    ("cli_setup.html", "https://docs.dask.org/en/latest/setup/cli.html"),
```

Comment:
The `redirect_files` list in `docs/source/conf.py` is for when we move an entire page to a new location. Since we're just modifying a section here, we don't need to include an entry in `redirect_files` for this PR. 

```suggestion
```

Reply:
Ha my bad ! I removed it.